XCSnippetr/Class/ViewControllers/XCSMainWindowController.m

```diff
@@ -418,6 +418,13 @@ - (void)didSubmitSnippetWithError:(NSError *)error
 {
     if (!error) {
         [[NSSound soundNamed:kSystemSoundSuccess] play];
+        
+        if (self.snippet.url) {
+            NSPasteboard *pasteboard = [NSPasteboard generalPasteboard];
+            [pasteboard clearContents];
+            [pasteboard writeObjects:@[ self.snippet.url ]];
```

Comment:
Oh no no, absoluteString is a method on NSURL that converts NSURL into NSString. I don't use it anymore. Please see code above that instantiates NSURL. Besides, pasteboard accepts NSURL so no need to do any conversions.